substrait/from_substrait.cpp

```diff
@@ -21,6 +21,27 @@
 #include "google/protobuf/util/json_util.h"
 
 namespace duckdb {
+const std::unordered_map<std::string, std::string> SubstraitToDuckDB::function_names_remap = { 
```

Comment:
Should we have a constexpr for these maps?

Reply:
I looked into it, but unfortunately there is no constexpr constructor support for std::map